src/modules/endpoint/models/CategoriesMapModel.js

```diff
@@ -30,7 +30,7 @@ class CategoriesMapModel {
    * @return {CategoryModel | undefined} The category
    */
   findCategoryByPath (path: string): ?CategoryModel {
-    return this._categories.get(normalizePath(path))
+    return this._categories.get(decodeURIComponent(normalizePath(path)))
```

Comment:
should we just use these globally defined functions like `decodeURIComponent` or should these be wrapped in a module (for separation / testability purposes)?

Is there an eslint rule for things like that?

Reply:
What do you mean wrapped in a module? Define a composition of  those two methods?

I don't see advantages of a separate function module.

Reply:
Hmmm don't know^^ I am just a little bit surprised, that we just use globally defined functions, but I think that's alright^^

Reply:
Global functions are perfectly fine as long as they do not have side effects :)

Reply:
Like `fetch` :P